robot_skills/src/robot_skills/topological_planner.py

```diff
@@ -0,0 +1,72 @@
+# System
+from typing import List
+
+# ROS
+import tf2_ros
+
+# TU/e Robotics
+from topological_action_planner_msgs.srv import Plan, PlanRequest, PlanResponse
+from topological_action_planner_msgs.msg import Edge, Node
+
+# Robot skills
+from .robot_part import RobotPart
+
+
+class TopologicalPlanner(RobotPart):
```

Comment:
#TODO add some explanation on the working principle of the topological action planner. At the very least a reference to  the relevant wikipedia article